react/react1/week3/todo-app-week3/src/components/TodoList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import TodoRow from './TodoRow';
+
+const TodoList = ({ todos, setTodos }) => {
+  return (
+    <div className='todo-container'>
+      <p className='todo-list'>
+        {todos.map((item) => (
```

Comment:
this is a very minor detail but just for better readability instead of having item as the paramater i would specify todo instead as it is more descriptive of what the paramater is